### DISCUSSION TITLE
sélection semaine sur créneau type

### DISCUSSION THREAD

**Discussion by MattBHub** [elefan-grenoble/gestion-compte#678] [Q&A] [answered]
Bonjour à tous, 

je suis en train de mettre à jour l'appli pour la Geg de Nancy (on est passablement en retard).
Dans la création d'un nouveau créneau type, je ne vois plus le choix de la semaine (A,B,C,D) .
Mais je vois qu'on peut filtrer par semaine pour rechercher des créneaux ...
Qqn peut-il me dire où ça se défini maintenant svp ?

**Comment by MattBHub** [comment, marked as answer]
ok, j'ai compris, c'est à l'édition du créneau que l'on peut ajouter des postes sur des cycles spécifiques
Pas bête comme fonctionnement.
A je vais continuer à tester du coup

**Comment by raphodn** [comment]
Salut !

en effet ca peut prêter à confusion.
on pourrait rajouter un message d'info/aide dans le formulaire de création.

Captures d'écran pour ceux qui passeraient par là :point_down: 

|||
|---|---|
|Nouveau créneau type|![Screenshot from 2022-12-23 15-00-25](https://user-images.githubusercontent.com/7147385/209348008-c9a24f5f-e2d7-4982-96f4-fa354d6befa6.png)|
|Editer le créneau type|![Screenshot from 2022-12-23 15-01-27](https://user-images.githubusercontent.com/7147385/209348206-8c6ad579-539c-40b9-bf3f-49e812d1cb5f.png)|
